Yoast/javascript
# Implement french stemmer

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Adds and refactors an external stemmer for French morphology together with a spec file.

## Relevant technical choices:

* A gold standard list based on an existing sample vocabulary was added. In its current form, the stems of this list only represent the output of the stemmer. They have not been checked manually.
* The words in `stemSpec` don't cover all patterns covered by the stemmer. Initially, that was the idea, but this approach was superseded by the coverage test. In the end, we left the specs we had already added in place and just added some very generic specs for a noun/verb/adjective.

## Test instructions

This PR can be tested by following these steps:

* Run the coverage test and check that there are no errors.
* Run stemSpec and check that there are no errors.

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/browse/LIN-367
